runelite-client/src/main/java/net/runelite/client/plugins/reportbutton/ReportButtonPlugin.java

```diff
@@ -61,6 +62,8 @@
 	private static final DateTimeFormatter DATE_TIME_FORMAT = DateTimeFormatter.ofLocalizedTime(FormatStyle.MEDIUM);
 	private static final DateFormat DATE_FORMAT = new SimpleDateFormat("MMM. dd, yyyy");
 
+	private int tickCounter;
```

Comment:
Please put this inbetween ``loginTime`` and ``ready`` and without newlines like so:
```java
private Instant loginTime;
private int tickCounter;
private boolean ready;
```

Reply:
Maybe it should also be renamed to something along the lines of ``ticksSinceLogin`` to be more clear about what it does.